src/app/utilities/transmart-utilities/transmart-packer-mapper.ts

```diff
@@ -20,20 +21,20 @@ const statusDictionary: { [status: string]: string | undefined } = {
 
 export class TransmartPackerMapper {
 
-  static mapCustomExportJob(job: TransmartPackerJob): ExportJob {
-    let ej = new ExportJob();
+  static mapCustomExportJob(job: TransmartPackerJob): TransmartExportJob {
```

Comment:
hmm, so it's mapped from `TransmartPackerJob` to `TransmartExportJob` and then to `ExportJob`? Is this necessary?

Reply:
for the architecture that we have now, yes, since transmartResourceSerivce now acts as a switch, and the export job it outputs would be 'transmartExportJob'. but indeed, in future if we are adopting essentially three endpoints, 1.transmart, 2.packer and 3.fractalis, we need to rethink the way we organize resourceService, fractalisService and transamrt-related services.